### PR TITLE
Unit tests for entities

### PR DESCRIPTION
**Pull request by PepiPetrov**
Added unit tests for entities Beneficiary and Campaign type.

### CODE REVIEW

**Comment by imilchev** [comment]
We agreed to not mock prisma with the others. Check with your colleagues on how to write tests using the real database.

**Comment by imilchev** [comment]
What is the status of this PR? I see that there are still mocks

**Comment by PepiPetrov** [comment]
@imilchev This is because I made them like @borislavstoychev's tests.

**Review by kachar — APPROVED** [review]
@imilchev I say let's move on with the mocked tests

As long as they are green for the controller and the service, I think we're good to merge it

File: apps/api/src/campaign-types/campaign-types.controller.spec.ts [hunk 1]
```diff
@@ -1,21 +1,159 @@
...
+      expect(result).toHaveLength(2)
+      expect(mockCampaignTypeService.remove).toBeCalledWith('0846e9cb-0668-448b-96de-1f35dfa9a1d4')
+    })
+
+    it('should remove many items', async () => {
+      const idsToDelete = [
+        '0846e9cb-0668-448b-96de-1f35dfa9a1d4',
+        '32f16697-db01-4ae4-8269-b0d7616f9820',
+      ]
+      const result = await controller.deleteMany({ ids: idsToDelete as [string] })
```
Comment (imilchev) [inline apps/api/src/campaign-types/campaign-types.controller.spec.ts:143]: why is there a function to delete multiple items? that is not needed. Can we delete it? @kachar what do you thing?

Comment (imilchev) [inline apps/api/src/campaign-types/campaign-types.controller.spec.ts:143]: why is there a function to delete multiple items? that is not needed. Can we delete it? @kachar what do you thing?

File: migrations/20220329134343_added_beneficiary_photo_role/migration.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+-- AlterEnum
+ALTER TYPE "campaign_file_role" ADD VALUE 'beneficiaryPhoto';
```
Comment (imilchev) [inline migrations/20220329134343_added_beneficiary_photo_role/migration.sql:2]: why is there a migration in a PR for tests? what happened here?

**Review by imilchev — CHANGES_REQUESTED** [review]

File: apps/api/src/campaign-types/campaign-types.controller.spec.ts (hunk 1 above)
Comment (kachar) [reply apps/api/src/campaign-types/campaign-types.controller.spec.ts:143]: We can delete it from the controller as we will not support multiple deletions at this point

File: apps/api/src/campaign/campaign.service.ts [hunk 3]
```diff
@@ -143,8 +143,7 @@ export class CampaignService {
     })
 
     donations.map((donation) => {
-      if (!donation.person)
-        donation.person = { firstName: 'anonymous', lastName: '' }
+      if (!donation.person) donation.person = { firstName: 'anonymous', lastName: '' }
```
Comment (kachar) [inline apps/api/src/campaign/campaign.service.ts:146]:
```suggestion
      if (!donation.person) {
        donation.person = { firstName: 'anonymous', lastName: '' }
      }
```

**Pull request closed without merging**